### ISSUE TITLE
Test Dataset Label

### ISSUE THREAD

**Issue by SeongES** [google-research-datasets/tpu_graphs#11]
Hi, I'm wondering if there are any plans to release the test dataset labels now that the Kaggle competition has ended. Thanks!

**Comment by sigeisler** [comment]
Hi @mangpo @samihaija any update on releasing the test labels? Or do I simply not find the reference? @SeongES did you find the test labels elsewhere?

**Comment by mangpo** [comment]
No, we didn't release the test labels (or have any plan to do so).
